code/Database.md

```diff
@@ -46,15 +46,15 @@ _Database interfaces may implement domain specific concerns, if there's a signif
 
 Each master database has an implementation. The master database and sub-databases are located in `<peer>/<peer>db`. As an example implementation for `satellite.DB` interface is at `satellite/satellitedb/*.go`.
 
-Often the databases have in-memory variants, which are useful for testing. They can also support different backends, such as SQLite and Postgres. For satellite the function `satellite/satellitedb/satellitedbtest.Run` loops through different backends making testing easier. Storage Node has a similar function.
+Often the databases have in-memory variants, which are useful for testing. They can also support different backends, such as SQLite and Postgres. For satellite the function `satellite/satellitedb/satellitedbtest#Run` loops through different backends making testing easier. Storage Node has a similar function.
```

Comment:
I tend to use `#` for referencing things inside of a package because for the `.` I use for referencing methods.

But if `.` is preferred, I'll rollback this change.

Reply:
Yeah, it's based on how Go prints the full name, rather than some particular editor.

Reply:
Btw, there wasn't set by my editor, it was my change